### DISCUSSION TITLE
8月14日支付的蓝灯专业版一年费用，今天却登录不上了也收不到验证码

### DISCUSSION THREAD

**Discussion by SHUO3166** [getlantern/lantern-forum-cn#809] [Q&A] [answered]
我的蓝灯账户是180635956@qq.com 订单号是2024081422001322841457487278 商家订单号是src 1PnbEpHGmABNLbfYCoehrB4F 如果国内对邮箱有限制，请把我的账号转为yangshuo3166@gmail.com

**Comment by SHUO3166** [comment]
![微信图片_20240824204630](https://github.com/user-attachments/assets/b9c219e5-de1b-43d8-a1db-857872c90db8)
付款截图如上

**Reply by wander0324** [reply]
亲，遇到同样的问题，请问你的问题解决了吗？

**Reply by SHUO3166** [reply]
目前还没人回复我

**Reply by MaxMKMax** [reply]
@SHUO3166 好的马上帮您安排! 不好意思周末不上线.

**Reply by MaxMKMax** [reply]
@wander0324 亲. 您好. 请您提供我支付单据和账号邮箱.

**Reply by wander0324** [reply]
> @wander0324 亲. 您好. 请您提供我支付单据和账号邮箱.

已发您邮箱，请查收，谢谢！账号邮箱85423166@qq.com

**Comment by SHUO3166** [comment]
@Mike777Khoo .

**Reply by MaxMKMax** [reply]
@SHUO3166 好的马上帮您安排! 不好意思周末不上线.

**Reply by SHUO3166** [reply]
您看下上面的信息，有账号和付款记录，谢谢

**Comment by wander0324** [comment]
***@***.***，支付单据请查看下图，谢谢！





------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年8月27日(星期二) 凌晨0:37
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 8月14日支付的蓝灯专业版一年费用，今天却登录不上了也收不到验证码 (Discussion #809)





 
@wander0324 亲. 您好. 请您提供我支付单据和账号邮箱.

**Comment by MaxMKMax** [comment, marked as answer]
@SHUO3166 您好. 技术人员给您更改好邮件了.

**Reply by wander0324** [reply]
> @SHUO3166 您好. 技术人员给您更改好邮件了.

请帮忙处理一下我的问题，谢谢！账号邮箱85423166@qq.com，支付截图已邮箱发给你的
![付款记录](https://github.com/user-attachments/assets/1bbda683-4ac1-41cd-b469-9feefdcf3d33)

**Reply by SHUO3166** [reply]
已经可以正常登录了，谢谢
